JS/ViewControllers/libraryChooseMediaLanguageViewController.js

```diff
@@ -0,0 +1,118 @@
+class LibraryChooseMediaLanguageViewController extends ViewController{
+
+    constructor() {
+        super();
+        this.shortLanguage;
+        this.language;
+        this.availableLanguages;
+    }
+
+    renderHtml(html) {
+        const htmlView = `
+            <section id="LibraryChooseMediaLanguageViewController" class="container">
+                <div class="row row-100 m12">
+                    <div class="col s12">
+                        <h1 class="center">Choose media language</h1>
+                    </div>
+                </div>
+                <div class="row row-25 m12">
+                    <form action="#">
+                        <select id="select-language" class="browser-default">
+                            <option value="none" selected></option>
+                        </select>
+                    </form>
+                </div>
+                <p id="demo"></p>
+                <div class="row row-50">
+                    <div class="col s12 center">
+                        <a id="choose-button" class="waves-effect waves-light btn-large disabled">choose</a>
+                    </div>
+                </div>
+                <div class="row row-100">
+                    <div class="col s12">
+                        <a class="btn-small right" href="index.html">Back to my menu</a>
+                    </div>
+                </div>
+            </section>
+        `;
+        super.renderHtml(htmlView);
+      }
+
+      setupProperties() {
+        this.languagePicker = $('#select-language');
+        this.chooseButton = $('#choose-button');
+        this.getAvailableLanguage();
```

Comment:
Nemal by sa tento riadok nachadzat v override metode `viewDidLoad()`? je to metoda ktora sluzi na nejake additional veci ktore sa maju stat ked sa vsetko nacita